nuxeo-features/nuxeo-elasticsearch/nuxeo-elasticsearch-audit/src/test/java/org/nuxeo/elasticsearch/TestESHistoryProvider.java

```diff
@@ -108,69 +297,60 @@ protected void createTestEntries() throws Exception {
 
         // wait at least 1s to be sure we have a precise timestamp in all DB
         // backend
-        Thread.sleep(500);
+        MILLISECONDS.sleep(500);
```

Comment:
Why this change? We're used to use Thread#sleep only, and for new usage we recommend awaitility which has the benefit to handle correctly the thread waiting and have an easy to use API. Could you rollback or use the new way?